jsonschema/WorkloadHorizontalValues.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$ref": "#/definitions/WorkloadHorizontalValues",
+    "definitions": {
+        "WorkloadHorizontalData": {
```

Comment:
There seem to be multiple schema def's duplicated - some in the same files as their "parent" model, some in their own. Should these be consolidated?